### ISSUE TITLE
New example: display a point by geocoded address

### ISSUE THREAD

**Issue by stevage** [mapbox/mapbox-gl-js#5030]
There wasn't an example of using the Geocoder API directly, without the Geocoder control. Perhaps you'd like to add something [like this](https://codepen.io/stevebennett/pen/xrQMMP).
```
<html>
<head>
    <meta charset='utf-8' />
    <title></title>
    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.js'></script>
    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.38.0/mapbox-gl.css' rel='stylesheet' />
    <!-- The magic of unpkg lets us use a module designed for NPM directly in the browser. -->
    <script src='https://unpkg.com/mapbox@1.0.0-beta7/dist/mapbox-sdk.min.js'></script>
    <style>
        body { margin:0; padding:0; }
        #map { position:absolute; top:0; bottom:0; width:100%; }
        h2 { background:#ff9;padding:0.5em;border:1px solid gray; border-radius:1em; }
    </style>
</head>
<body>

<div id='map'></div>
<script>
mapboxgl.accessToken = 'pk.eyJ1IjoiY2l0eW9mbWVsYm91cm5lIiwiYSI6ImNpejdob2J0czAwOWQzM21ubGt6MDVqaHoifQ.55YbqeTHWMK_b6CEAmoUlA';

var query = "Sydney Opera House";

var mapboxClient = new MapboxClient(mapboxgl.accessToken);
mapboxClient.geocodeForward(query, { limit: 1 }, function(err, res) { 

    var queryCoords = res.features[0].geometry.coordinates;

    var map = new mapboxgl.Map({
        container: 'map', // container id
        style: 'mapbox://styles/mapbox/streets-v9', //stylesheet location
        center: queryCoords,
        zoom: 14 // starting zoom
    });


    var div = document.createElement('div');
    div.innerHTML = '<h2>' + res.features[0].text + '</h2>';

    new mapboxgl.Marker(div)
        .setLngLat(queryCoords)
        .addTo(map);
}); 
</script>

</body>
</html>
```

**Comment by andrewharvey** [comment]
https://stackoverflow.com/a/48087483/6702659 is another example of where this example would help.

**Comment by andrewharvey** [comment]
closed by #7152
